### DISCUSSION TITLE
ScriptReference/Physics.ComputePenetration

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#240] [Notes]
# ScriptReference/Physics.ComputePenetration

https://docs.unity3d.com/ScriptReference/Physics.ComputePenetration.html

**Comment by mvi** [comment]
Note you can easily wrap this with a simpler method signature to create a simple collider-collider intersection test, e.g.  
  

```csharp
public static bool Intersects(this BoxCollider a, Collider b)  
{  
    return Physics.ComputePenetration(a, a.transform.position, a.transform.rotation, b, b.transform.position, b.transform.rotation, out _, out _);  
}
```
